Proposed abstract & title for pathogenesis manuscript (#746)
greenelab/covid19-review: content/07.pathogenesis.md

> ## Genomic and Viral Structure of SARS-CoV-2 in the Context of Pathogenesis, Symptomology, and Transmission

Reviewer: Title is great!

### Abstract

The novel coronavirus SARS-CoV-2, which emerged in late 2019, has been a major force shaping the year 2020 as it spread around the world infecting tens of millions of people with coronavirus disease 2019 (COVID-19).
While the viral species was unknown prior to January 2020, its similarity to other coronaviruses that infect humans has allowed for rapid insight into the mechanisms that it uses to infect human hosts, as well as the ways in which the human immune system can respond.
Here, we contextualize SARS-CoV-2 among other coronaviruses and identify what is known and what can be inferred about its behavior once inside a human host.

> Because the genomic content of coronaviruses is highly conserved and specifies the virus's structure, early genomic analysis provided a significant head start in predicting viral pathogenesis.

Reviewer 2: ```suggestion
Because the genomic content of coronaviruses, which specifies the virus's structure, is highly conserved, early genomic analysis provided a significant head start in predicting viral pathogenesis.
```

> The pathogenesis of the virus can then offer insight into symptomology, transmission, and individual susceptibility.

Reviewer: ```suggestion
The pathogenesis of the virus offers insights into symptomology, transmission, and individual susceptibility.
```

Reviewer: I may be wrong there

Additionally, prior research into interactions between the human immune system and coronaviruses identifies how these viruses can evade destruction.
We also explore systems-level research into the regulatory and proteomic effects of SARS-CoV-2 infection and the immune response.
Understanding the structure and behavior of the virus serves to contextualize many facets of the COVID-19 pandemic and can influence on efforts to control the virus and treat the disease.

### Importance

COVID-19 involves a number of organ systems and can present with a wide range of symptoms.
Understanding how the virus infects epithelial cells, however, serves to contextualize how these systems connect.

> Similarly, the modes of viral transmission have been under debate throughout much of 2020, yet the available research suggests that these patterns are very similar to those seen with SARS-CoV and possibly MERS-CoV.

Reviewer: ```suggestion
Similarly, the modes of viral transmission have been under debate throughout much of 2020, yet the available research suggests that these patterns are very similar to those seen with the closely related SARS-CoV and possibly MERS-CoV.
```

> Similarly, the modes of viral transmission have been under debate throughout much of 2020, yet the available research suggests that these patterns are very similar to those seen with the closely related SARS-CoV and possibly MERS-CoV.

Author: ```suggestion
Similarly, the modes of viral transmission have been under debate throughout much of 2020, yet the available research suggests that these patterns are very similar to those seen with the closely related viruses SARS-CoV-1 and possibly MERS-CoV.
```

## Genomic and Viral Structure of SARS-CoV-2 in the Context of Pathogenesis, Symptomology, and Transmission

### Abstract

The novel coronavirus SARS-CoV-2, which emerged in late 2019, has been a major force shaping the year 2020 as it spread around the world infecting tens of millions of people with coronavirus disease 2019 (COVID-19).
While the viral species was unknown prior to January 2020, its similarity to other coronaviruses that infect humans has allowed for rapid insight into the mechanisms that it uses to infect human hosts, as well as the ways in which the human immune system can respond.
Here, we contextualize SARS-CoV-2 among other coronaviruses and identify what is known and what can be inferred about its behavior once inside a human host.
Because the genomic content of coronaviruses is highly conserved and specifies the virus's structure, early genomic analysis provided a significant head start in predicting viral pathogenesis.
The pathogenesis of the virus offers insights into symptomology, transmission, and individual susceptibility.

> Additionally, prior research into interactions between the human immune system and coronaviruses identifies how these viruses can evade destruction.

Reviewer 2: Just checking that this is meant to say "destruction" and not "detection", both work

Author: ```suggestion
Additionally, prior research into interactions between the human immune system and coronaviruses identifies how these viruses can evade the immune system's protective mechanisms.
```
Haha, they both sound weird now that you point it out! Suggesting a different route!

We also explore systems-level research into the regulatory and proteomic effects of SARS-CoV-2 infection and the immune response.

> Understanding the structure and behavior of the virus serves to contextualize many facets of the COVID-19 pandemic and can influence on efforts to control the virus and treat the disease.

Reviewer 2: ```suggestion
Understanding the structure and behavior of the virus serves to contextualize the many facets of the COVID-19 pandemic and can influence efforts to control the virus and treat the disease.
```
